WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AIToneVoice/AIToneVoiceView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+
+/// View to select AI tone and voice
+///
+struct AIToneVoiceView: View {
+    @StateObject private var viewModel: AIToneVoiceViewModel
+
+    @Environment(\.presentationMode) private var presentation
+
+    init(viewModel: AIToneVoiceViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
```

Comment:
❓ Why do we need to use `StateObject` here instead of `ObservedObject`?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10754/commits/007e910a4fbe343bc3734ab22df2ceb7d9a45f36